admin/spec/back-end/lib/validator/school-impersonation/school-impersonation-empty-value-validator.spec.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+/* global describe, it expect */
+const schoolImpersonationEmptyValueValidator = require('../../../../../lib/validator/school-impersonation/school-impersonation-empty-value-validator')
+
+describe('schoolImpersonationEmptyValueValidator', function () {
+  it('should not add a validation error if dfeNumber is not empty', function () {
+    const dfeNumber = '1230000'
```

Comment:
When do you convert the form input (string) into the required type for the DB lookup (number)?  Don't you also need to validate the post-conversion result to ensure it is an integer?

Reply:
@jon-shipley resolving this as the caller service implementation and naming has changed.